test/table/test_table_padding.py

```diff
@@ -12,9 +12,9 @@
 HERE = Path(__file__).resolve().parent
 
 
-def run_comparison(pdf, name, tmp_path):
+def run_comparison(pdf, name, tmp_path, generate=False):
     filename = HERE / f"{name}.pdf"
-    assert_pdf_equal(pdf, filename, tmp_path, generate=False)
+    assert_pdf_equal(pdf, filename, tmp_path, generate=generate)
```

Comment:
I don't see a reason to leave the generate parameter here but it's not a showstopper.